dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/HVDC/ACEmulationOuterControl.mo

```diff
@@ -0,0 +1,63 @@
+within Dynawo.Electrical.Controls.HVDC;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model ACEmulationOuterControl "AC Emulation outer control for HVDC"
+
+  Connectors.ImPin PRefSet1RawPu (value(start = PRefSet10Pu)) "Raw reference active power of HVDC link 1 in p.u (base SnRef)";
+  Connectors.ImPin PRefSet2RawPu (value(start = PRefSet20Pu)) "Raw reference active power of HVDC link 2 in p.u (base SnRef)";
+
+  Connectors.ImPin KACEmulation1Raw (value(start = KACEmulation10)) "Raw inverse of the emulated AC reactance for HVDC link 1";
+  Connectors.ImPin KACEmulation2Raw (value(start = KACEmulation20)) "Raw inverse of the emulated AC reactance for HVDC link 2";
+
+  Connectors.BPin running1 (value(start = true)) "Boolean assessing if the HVDC link 1 is running";
+  Connectors.BPin running2 (value(start = true)) "Boolean assessing if the HVDC link 2 is running";
+
+  parameter Types.PerUnit KACEmulation10 "Start value of inverse of the emulated AC reactance for HVDC link 1";
+  parameter Types.PerUnit KACEmulation20 "Start value of inverse of the emulated AC reactance for HVDC link 2";
+  parameter Types.ActivePowerPu PRefSet10Pu "Start value of reference active power in p.u (base SnRef) for HVDC link 1";
+  parameter Types.ActivePowerPu PRefSet20Pu "Start value of reference active power in p.u (base SnRef) for HVDC link 2";
+
+  output Types.ActivePowerPu PRefSet1Pu (start = PRefSet10Pu) "Reference active power of HVDC link 1 in p.u (base SnRef)";
+  output Types.ActivePowerPu PRefSet2Pu (start = PRefSet20Pu) "Reference active power of HVDC link 2 in p.u (base SnRef)";
+
+  output Types.PerUnit KACEmulation1 (start = KACEmulation10) "Inverse of the emulated AC reactance for HVDC link 1";
```

Comment:
I reopen the question about using input and output keyword:  I remember we said not to use them once but maybe it has evolved. I personally have no opinion.

Reply:
> e aspect of transferring the power to another HVDC.

I agree the name could be more explicit concerning the active power transfer.
We could have a model with only the power references, and a model extending the first one and adding the KAC equation. The first one could be used the Setpoints-HVDC and the second one with ACEmulation-HVDC. Is this what you had in mind?

Reply:
I have no preference either on wether using an output or a "simple" variable.

Reply:
I used connectors instead of output